pkg/email/mailgun.go

```diff
@@ -0,0 +1,156 @@
+package email
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"html/template"
+	"io/ioutil"
+	"strings"
+	"time"
+
+	"github.com/juju/errors"
+	"github.com/mailgun/mailgun-go/v4"
+	"gopkg.in/yaml.v2"
+)
+
+// MailgunProviderName is mailgun provider name
+const MailgunProviderName = "mailgun"
+
+const defaultHTMLTemplateForUserEmail = `
```

Comment:
i don't think the templates should be part of `email` package.
they need to be in `martini` service , or where we will create the message